lib/matplotlib/cbook/__init__.py

```diff
@@ -143,6 +144,11 @@ class CallbackRegistry:
         >>> callbacks.disconnect(id_eat)
```

Comment:
Since you're touching this, mind putting  a new line before `.disconnect` is introduced?